src/components/page/index.jsx

```diff
@@ -35,16 +43,16 @@ class Page extends React.PureComponent {
           </div>
         )}
         {/* </section> */}
-        {WidgetTag && (
-          <div className={styles.gridWidget}>
-            <WidgetTag {...this.props} />
-          </div>
-        )}
         {image && (
           <div className={styles.gridImage}>
             <img src={image.mediaPath} alt={image.altText} />
           </div>
         )}
+        {widgetRow === 'bottom' && WidgetTag && (
+          <div className={styles[`gridWidget${capitalize(widgetRow)}`]}>
+            <WidgetTag {...this.props} />
+          </div>
+        )}
```

Comment:
I think this is probably where we want a widget to end up by default so I think you could change line 51 to something like: `(widgetRow === 'bottom' || !widgetRow) && WidgetTag && `

I think...